tests/manage/rgw/conftest.py

```diff
@@ -15,15 +15,20 @@ def pytest_collection_modifyitems(items):
         items: list of collected tests
 
     """
+    specified_markers = config.getoption("-m")
     if (
-        config.ENV_DATA["platform"].lower() not in ON_PREM_PLATFORMS
+        ocs_ci_config.ENV_DATA["platform"].lower() not in ON_PREM_PLATFORMS
         or version.get_semantic_ocs_version_from_config() < version.VERSION_4_5
+        or "noobaa" in specified_markers  # Exclude RGW tests out of red_squad tests
     ):
+        log.info("Removing RGW tests from the collected items")
         for item in items.copy():
             if "manage/rgw" in str(item.fspath):
```

Comment:
This path will change

Reply:
Reverted in favor of explicit TC labeling